libs/ngx-material-intl-tel-input-lib/src/lib/ngx-material-intl-tel-input-lib/ngx-material-intl-tel-input-lib.component.scss

```
@@ -6,71 +6,140 @@
     display: flex;
     flex-direction: column;
     width: 100%;
+
     .main-label {
       padding: 2px 0;
       font-size: 14px;
     }
-    .tel-form {
-      display: flex;
-      justify-content: center;
-      align-items: center;
-      width: 100%;
-      border-radius: 8px;
-      border: 1px solid #d8d8d8;
-      background-color: #fbfbfb;
+
+    .tel-form:has(.mat-form-field-appearance-fill) {
+      border-style: solid;
+      border-width: var(--mdc-filled-tel-form-outline-width, 1px);
+      border-color: var(--mdc-filled-tel-form-outline-color, #d8d8d8);
+      border-radius: var(--mdc-filled-tel-form-container-shape, 4px);
+      background-color: var(
+        --mdc-filled-tel-form-background,
+        #fbfbfb
+      ) !important;
+
       &.is-focused {
-        border: 1px solid var(--mdc-theme-primary, #209ffc);
-        outline: 2px solid rgba(var(--mdc-theme-primary-rgb, 32, 159, 252), 0.3);
-        background-color: #fff;
+        border-color: var(--mdc-filled-tel-form-focus-outline-color, #209ffc);
+        outline: 2px solid
+          rgba(var(--mdc-filled-tel-form-focus-outline-color, #209ffc), 0.3);
```

```suggestion
          rgba(var(--mdc-filled-tel-form-focus-outline-color-rgb, var(--mdc-theme-primary-rgb, 32, 159, 252)), 0.3);
```
I guess this variable it's not in RGB format. Maybe it could be better to add a new one?